cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -19,10 +19,11 @@ package apiconfig
 import (
 	"fmt"
 
+	"k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1beta1"
+rbachelper "k8s.io/kubernetes/pkg/apis/rbac/v1beta1"
```

Comment:
@seh i couldn't find your other comment about exporting a function instead of SchemeBuilder in register.go, so i reply here. That will require additional changes to the code generators (deepcopy, conversion, etc), some (deepcopy-gen) are not in the kubernetes repo and will thus require multiple PRs, so i decided not to do that refactor in this PR.